hbase-handler/src/test/org/apache/hadoop/hive/hbase/TestHiveHBaseTableInputFormatV2.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.hbase;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+/**
+ *  Test tot make sure the HiveHBaseTableInputFormatV2 can ONLY assignable to old version
+ */
+public class TestHiveHBaseTableInputFormatV2 {
+
+  @Test
+  public void testInstanceOfTestHiveHBaseTableInputFormatV2() {
```

Comment:
Not very detailed tests :)

Reply:
For this class, I did not see many functions can be easily unit test, and I believe that`s why there was even no test for HiveHBaseTableInputFormat, do you have any suggestion on which function or how we can do more test?